ej/models.py

```diff
@@ -8,6 +8,8 @@
 class Problem(db.Entity):
     title = Required(str, unique=True)
     time_limit = Required(int)
+    custom_check = Required(bool)
```

Comment:
@simiaosimis Why do we need the extra **bool** field? I think one nullable **str** field as *custom_check* would be enough.